core/src/main/java/org/apache/iceberg/variants/VariantArray.java

```diff
@@ -20,6 +20,10 @@
 
 /** An variant array value. */
 public interface VariantArray extends VariantValue {
+  default int numElements() {
+    throw new UnsupportedOperationException();
```

Comment:
Has `VariantArray` been in a release?

Reply:
Just checked. It's not in any release. What is the question for?

Reply:
This has been added so I just rebased so we don't need this anymore.